hyperlink/test/common.py

```diff
@@ -0,0 +1,70 @@
+
+
+from unittest import TestCase
+
+
+class HyperlinkTestCase(TestCase):
```

Comment:
Intended only as a comment: would it be better to use py.test or add a conditional dependency on unittest2 than directly backport this code?